site/layouts/partials/version-banner.html

```diff
@@ -1,5 +1,5 @@
 
   {{ $color := "primary" }}
   <div class="pageinfo pageinfo-{{ $color }}">
-    This is the documentation for version 0.39. For versions >= 1.0.0-alpha.1, see <a href="https://kpt.dev/">kpt.dev</a>.
+    This is the documentation for version 0.39. For versions >= 1.0.0-alpha.1, see <a href="https://kpt.dev/installation/">kpt.dev</a>.
```

Comment:
to clarify, I think this is fine to point to kpt.dev. For drop-down menu on kpt.dev, we should link to installation pages.

Reply:
Reverted.